README.md

```diff
@@ -18,8 +18,8 @@ See our numerous [**examples and tutorials**](https://gpytorch.readthedocs.io/en
 ## Installation
 
 **Requirements**:
-- Python >= 3.6
-- PyTorch >= 1.9
+- Python >= 3.7
+- PyTorch >= 1.11
 
```

Comment:
Should add linear_operator here (the `main` github branch for now until we have made a proper release).

Reply:
We've made a proper release now, so it should be fine.